_quarto.yml

```
@@ -2,22 +2,28 @@ project:
   type: book
 
 book:
-  title: "Managing an open source project workshop"
+  title: "Managing an open source project"
```

I dropped the word workshop since it made the title quite long and the heading underneath says "Workshop Overview." I'm happy to change it back to the original if you prefer that! ⏬ 

```suggestion
  title: "Managing an open source project workshop"
```